### PR TITLE
PneumaticCleaning/ClimbFeature AutonColorWheel

### PR DESCRIPTION
**Pull request by mobamba7447**
So I setup The pneumatic subsystem to the point where we can continuously add more if needed. There are some changes with control. This is mainly for the auton spinner. The auton spinner has been created but it's experimental We need the limelight for full autonomous testing

### CODE REVIEW

File: src/main/java/frc/robot/commands/ColorSelectionCommand.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+public class ColorSelectionCommand extends Command {
+  final Color kBlueTarget = ColorMatch.makeColor(0, 255, 255);
+  final Color kGreenTarget = ColorMatch.makeColor(0, 255, 0);
+  final Color kRedTarget = ColorMatch.makeColor(255, 0, 0);
+  final Color kYellowTarget = ColorMatch.makeColor(255, 255, 0);
+  private final I2C.Port i2cPort = I2C.Port.kOnboard;
+  String colorString;
+  double counter = 0;
+  final ColorSensorV3 m_colorSensor = new ColorSensorV3(i2cPort);
+     final ColorMatch m_colorMatcher = new ColorMatch();
```
Comment (Involuta) [inline src/main/java/frc/robot/commands/ColorSelectionCommand.java:28]: why is there an indent here?

File: src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java [hunk 2]
```diff
@@ -17,24 +17,37 @@
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
 
-  private final DoubleSolenoid doubleSol = new DoubleSolenoid(0,1);
+  private final DoubleSolenoid doubleSolenoid = new DoubleSolenoid(0,1);
+  private final DoubleSolenoid doubleSolenoid2 = new DoubleSolenoid(2,3);
 
-
-  public void ShooterToggle(int solenoidToggle) {
+  public void ShooterToggle(final int solenoidToggle) {
```
Comment (Involuta) [inline src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java:23]: why did you add final here?

Comment (mobamba7447) [reply src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java:23]: because errors in the vscode

File: src/main/java/frc/robot/commands/ColorSelectionCommand.java (hunk 1 above)
Comment (mobamba7447) [reply src/main/java/frc/robot/commands/ColorSelectionCommand.java:28]: because errors in vscode

File: src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java [hunk 3]
```diff
@@ -17,24 +17,37 @@
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
 
-  private final DoubleSolenoid doubleSol = new DoubleSolenoid(0,1);
+  private final DoubleSolenoid doubleSolenoid = new DoubleSolenoid(0,1);
```
Comment (Involuta) [inline src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java:20]: why did you change the name to doubleSolenoid? doubleSol is shorter and cooler

File: src/main/java/frc/robot/commands/ColorSelectionCommand.java (hunk 1 above)
Comment (mobamba7447) [reply src/main/java/frc/robot/commands/ColorSelectionCommand.java:28]: this one makes no sense to me 
it's just being wack

File: src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java (hunk 3 above)
Comment (mobamba7447) [reply src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java:20]: no

Comment (mobamba7447) [reply src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java:20]: idk, i was lazy, I called climb solenoid double solenoid2. I

Comment (Involuta) [reply src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java:20]: : (

File: src/main/java/frc/robot/commands/ColorSelectionCommand.java (hunk 1 above)
Comment (Involuta) [reply src/main/java/frc/robot/commands/ColorSelectionCommand.java:28]: what did the error say?

File: src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java (hunk 3 above)
Comment (mobamba7447) [reply src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java:20]: pls just accept

File: src/main/java/frc/robot/commands/ColorSelectionCommand.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+
+public class ColorSelectionCommand extends Command {
+  final Color kBlueTarget = ColorMatch.makeColor(0, 255, 255);
+  final Color kGreenTarget = ColorMatch.makeColor(0, 255, 0);
+  final Color kRedTarget = ColorMatch.makeColor(255, 0, 0);
+  final Color kYellowTarget = ColorMatch.makeColor(255, 255, 0);
+  private final I2C.Port i2cPort = I2C.Port.kOnboard;
+  String colorString;
+  double counter = 0;
```
Comment (Involuta) [inline src/main/java/frc/robot/commands/ColorSelectionCommand.java:26]: why isn't counter an int

**Review by Involuta — APPROVED** [review]
bueno but can you rename the pneumatic shooter subsystem to pneumatic subsystem
